.github/workflows/pr-commands.yaml

```diff
@@ -112,7 +113,7 @@ jobs:
 
       - uses: r-lib/actions/pr-push@v2
         with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          repo-token: ${{ secrets.ZS_PAT }}
```

Comment:
Is this an access key for your actual account? I guess this really isn't ideal. Is it restricted to just this repo?

Reply:
It's a standard (not a new fine-grained) PAT linked to my identity but scoped only to public repos and read/write/push for code. I tried to set up a fine-grained token but those need to be issued by the org owning the repo rather than scoped to the individual.

I believe the scoping to only read/write for code is quite a bit more limited than the `GITHUB_TOKEN` automatically supplied to workflows. But because this token is linked to my identity it can trigger additional workflows after the push.

Reply:
> If we thought it was fine for CI to fail to run after the push we could use the

This becomes deeply annoying and is one of the reasons we switched to doing automated stuff via independent PRs vs in peoples PRs in epinowcast so I think this is all fine as you've set it up. 

Thanks for the extra detail.